src/System.Runtime/tests/TestLoadAssembly/TestLoadAssembly.csproj

```diff
@@ -13,11 +13,6 @@
   <ItemGroup>
     <Compile Include="TestLoadAssembly.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
```

Comment:
I removed it as we are able to build as netstandard. but I am going to work on enabling netfx vertical test runs so I may bring back what is needed at that time